File: output/llama_4_maverick/avatar/algo_based_translation/java/python/codeforces_682_B.py

def main():
    n = int(input())
    lst = list(map(int, input().split()))
    lst.sort()
    c = 1
    for i in lst:
        if i >= c:
            c += 1
    print(c - 1 if n > 0 and lst[0] > 1 else c)

if __name__ == "__main__":
    main()

File: output/deepseek_chat/avatar/algo_based_translation/python/java/successful_translation/codeforces_682_B.java
import java.util.*;
import java.io.*;

public class codeforces_682_B {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        String[] tokens = br.readLine().split(" ");
        int[] l = new int[tokens.length];
        for (int i = 0; i < tokens.length; i++) {
            l[i] = Integer.parseInt(tokens[i]);
        }
        int f = 1;
        Arrays.sort(l);
        int c = 0;
        for (int i : l) {
            if (i > c) {
                c++;
            }
        }
        System.out.println(c + 1);
    }
}
